utilities/transactions/write_committed_transaction_ts_test.cc

```diff
@@ -314,6 +327,9 @@ TEST_P(WriteCommittedTxnWithTsTest, RecoverFromWal) {
     s = GetFromDb(ReadOptions(), handles_[1], "key1", /*ts=*/24, &value);
     ASSERT_OK(s);
     ASSERT_EQ("value_3", value);
+
+    s = GetFromDb(ReadOptions(), handles_[1], "baz", /*ts=*/24, &value);
```

Comment:
Do you mean to also commit this transaction?

Reply:
oh, sorry for the confusion. This is to test the case that a recovered prepared transaction is not commited, and the key should not be found. It's kind of what the "foo" key in the original test does, I kind of shifted it down to the last one.